### DISCUSSION TITLE
New challenge ScrollArea

### DISCUSSION THREAD

**Discussion by efgomes** [shadcn-ui/ui#4589] [General]
"After numerous attempts, how can this yellow div be set to have a height of 100%?"

	<Card className="h-full p-4 w-full">
			<ScrollArea className="h-full bg-green-200">
				<div className="h-full bg-yellow-100">
					essa div tem que ficar em 100%
				</div>
				<ScrollBar orientation="horizontal" />
			</ScrollArea>
		</Card>
		
		
![image](https://github.com/user-attachments/assets/77087f4b-26b0-43a3-88b3-5215bea22d72)

**Comment by Earthyyy** [comment]
Check the fix from this [PR](https://github.com/shadcn-ui/ui/pull/4498). It worked for me
